packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -148,14 +136,25 @@ const TradersHub = () => {
                         ) : (
                             <ButtonToggleLoader />
                         )}
-                        {selected_platform_type === 'options' && <OptionsAndMultipliersListing />}
-                        {selected_platform_type === 'cfd' && <CFDsListing />}
+                        {renderOrderedPlatformSections(
+                            selected_platform_type === 'cfd',
+                            selected_platform_type === 'options'
+                        )}
                     </MobileWrapper>
                     <ModalManager />
                     {scrolled && <TourGuide />}
                 </div>
             </Div100vhContainer>
-            {is_eu_low_risk && <EUDisclamer />}
+            {is_eu_low_risk && (
+                <div className='disclamer'>
+                    <Text align='left' className='disclamer-text' size={isMobile() ? 'xxxs' : 'xs'}>
```

Comment:
**NIT:** Recently came across is_mobile that is present as part of ui store. If this component supports useStore hooks, better to use`is_mobile` instead of `isMobile()` as the flag gets tiggered on re-sizing

Reply:
@likhith-deriv this is replaced with is_mobile from ui-store.